tools/test-js-with-node

```diff
@@ -195,6 +195,8 @@ EXEMPT_FILES = make_set(
         "web/src/resize.ts",
         "web/src/resize_handler.ts",
         "web/src/rows.ts",
+        "web/src/saved_snippets.ts",
+        "web/src/saved_snippets_ui.ts",
```

Comment:
Usually I'd expect it to be easy to have 100% test coverage on the non-UI module.